mapstore_test.go

```diff
@@ -1,46 +1,46 @@
 package smt
 
-import(
-    "bytes"
-    "crypto/sha256"
-    "testing"
+import (
+	"bytes"
+	"crypto/sha256"
+	"testing"
 )
 
 func TestSimpleMap(t *testing.T) {
-    sm := NewSimpleMap()
-    h := sha256.New()
-    var value []byte
-    var err error
+	sm := NewSimpleMap()
+	h := sha256.New()
+	var value []byte
+	var err error
 
-    h.Write([]byte("test"))
+	h.Write([]byte("test"))
 
-    // Tests for Get
-    _, err = sm.Get(h.Sum(nil))
-    if err == nil {
-        t.Error("did not return an error when getting a non-existent key")
-    }
+	// Tests for Get
+	_, err = sm.Get(h.Sum(nil))
+	if err == nil {
+		t.Error("did not return an error when getting a non-existent key")
+	}
 
-    // Tests for Put
-    err = sm.Put(h.Sum(nil), []byte("hello"))
-    if err != nil {
-        t.Error("updating a key returned an error")
-    }
-    value, err = sm.Get(h.Sum(nil))
-    if bytes.Compare(value, []byte("hello")) != 0 {
-        t.Error("failed to update key")
-    }
+	// Tests for Put
+	err = sm.Put(h.Sum(nil), []byte("hello"))
+	if err != nil {
+		t.Error("updating a key returned an error")
+	}
+	value, err = sm.Get(h.Sum(nil))
```

Comment:
No biggie as this is "just" test code but: this `err` is never read (either `_` ignore it or handle it). This is sth that a linter would catch as well.